frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -102,10 +106,10 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
   };
 
   return (
-    <Tooltip
-      role="none"
-      data-testid="model-status-tooltip"
-      content={
+    <Popover
+      data-testid="model-status-popover"
+      position="right"
```

Comment:
Why default the position to the right? makes more sense to default it upwards as to not overlap the row.